src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorSymbolReferenceFinder.cs

```diff
@@ -98,22 +98,17 @@ private static bool IsPotentialReference(PredefinedType predefinedType, ISyntaxF
         CancellationToken cancellationToken)
     {
         // First just look for this normal constructor references using the name of it's containing type.
-        var name = methodSymbol.ContainingType.Name;
+        var containingType = methodSymbol.ContainingType;
+        var containingTypeName = containingType.Name;
         AddReferencesInDocumentWorker(
-            methodSymbol, name, state, processResult, processResultData, cancellationToken);
+            methodSymbol, containingTypeName, state, processResult, processResultData, cancellationToken);
 
         // Next, look for constructor references through a global alias to our containing type.
         foreach (var globalAlias in state.GlobalAliases)
```

Comment:
we both want to search for references to global aliases anywhere that might be to our symbol, and...